packages/webview_flutter/ios/Tests/FLTWebViewTests.m

```diff
@@ -64,4 +66,22 @@ - (void)webViewScrollIndicatorAticautomaticallyAdjustsScrollIndicatorInsetsShoul
   }
 }
 
+- (void)testContentInsetsSumAlwaysZeroAfterSetFrame {
+  FLTWKWebView *webView = [[FLTWKWebView alloc] initWithFrame:CGRectMake(0, 0, 300, 400)];
+  webView.scrollView.contentInset = UIEdgeInsetsMake(0, 0, 300, 0);
+  XCTAssertFalse(UIEdgeInsetsEqualToEdgeInsets(webView.scrollView.contentInset, UIEdgeInsetsZero));
+  webView.frame = CGRectMake(0, 0, 300, 200);
+  XCTAssertTrue(UIEdgeInsetsEqualToEdgeInsets(webView.scrollView.contentInset, UIEdgeInsetsZero));
+
+  if (@available(iOS 11, *)) {
```

Comment:
What's the effect of such a condition on a XCTest, are we running once with iOS < 11 and once with iOS >= 11?

Reply:
our ci is currently running on the latest iOS (13), I added this condition in case we decided to run CI for a lower iOS version. The current CI won't run the < iOS 11 path.